src/models/extension.ts

```diff
@@ -2,6 +2,5 @@ import type { BaseModel } from './base';
 
 export interface ExtensionInterface<T = any> extends BaseModel {
   id(): string;
-  version(): string;
```

Comment:
Please ensure this change is proposed in https://github.com/asyncapi/parser-api/pull/71

Reply:
Done https://github.com/asyncapi/parser-api/pull/71#discussion_r1006894610 :)